packages/plugins/src/SidebarPlugin.ts

```diff
@@ -164,15 +152,29 @@ const SidebarPlugin: PluginType<SidebarPluginPage, SidebarPluginOptions, Sidebar
         deep: sidebarRootLevel
       });
 
+      //Map into Sidebar Groups and sort children according to priority, priority = 1 has the highest priority
+      function sortSidebarGroups(sidebarData) {
+        const sortedGroupedPages = sidebarData.map(page => {
+          if (page.childNodes) {
+            const sortedPages = page.childNodes.sort((a, b) => a.priority - b.priority);
```

Comment:
Priority = 1 now has the highest priority. Think this is most intuitive and it aligns with how the priority ranking for the plugins work